README.md

```diff
@@ -11,17 +26,15 @@ Micro-service to save/load fiddles for [NEAR Studio](https://github.com/NEARProt
 brew install postgres
 ```
 
-
-## Run database 
+### Run database 
 
 ```
 brew services start postgresql
 createuser fiddle -W  # It'll ask for password, enter "fiddle"
-createdb -O fiddle fiddle_development
-createdb -O fiddle fiddle_test
```

Comment:
for local development it makes sense to have different DBs for development and test, cause otherwise running tests is going to wipe development DB which you don't want to wipe